Protocol.playground/Contents.swift

```diff
@@ -0,0 +1,30 @@
+protocol ShoppingList {
+    /// Stores items to shop
+    var shoppingList: [String] { get }
+    /// Max number of items
```

Comment:
```suggestion
    /// Maximum number of items
```

Don't be afraid of using full words.

Reply:
I even suggest:
```suggestion
    /// Maximum number of items that can be stored in the list. 
```

Reply:
Done. Thank you for your comment.